<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>FLUIDSynthGuiBase</class>
 <widget class="QDialog" name="FLUIDSynthGuiBase" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>514</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>FLUID Synth</string>
  </property>
  <property name="windowIcon" >
   <iconset/>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>8</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame_2" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="loadedFontsLabel" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>&lt;span style=&quot; font-weight:600;&quot;>Loaded Soundfonts&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="textFormat" >
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="sfTreeWidget" >
        <property name="columnCount" >
         <number>2</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>0</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1" >
    <widget class="QFrame" name="frame_3" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="fontSetupLabel" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>&lt;span style=&quot; font-weight:600;&quot;>Channel Setup&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
        </property>
        <property name="alignment" >
         <set>Qt::AlignHCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTreeWidget" name="channelTreeWidget" >
        <property name="columnCount" >
         <number>3</number>
        </property>
        <property name="sortingEnabled" >
         <bool>true</bool>
        </property>
        <column>
         <property name="text" >
          <string>0</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>1</string>
         </property>
        </column>
        <column>
         <property name="text" >
          <string>2</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item rowspan="2" row="1" column="1" >
    <widget class="QFrame" name="ChorusFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="Chorus" >
        <property name="text" >
         <string>Chorus</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ChorusNumberLabel" >
          <property name="text" >
           <string>Number</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="ChorusNumber" >
          <property name="maximum" >
           <number>127</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="ChorusTypeLabel" >
          <property name="text" >
           <string>Type</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="ChorusType" >
          <item>
           <property name="text" >
            <string>Sine</string>
           </property>
          </item>
          <item>
           <property name="text" >
            <string>Triangle</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ChorusSpeedLabel" >
          <property name="text" >
           <string>Speed</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ChorusSpeed" >
          <property name="maximum" >
           <number>16383</number>
          </property>
          <property name="singleStep" >
           <number>16</number>
          </property>
          <property name="pageStep" >
           <number>1638</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval" >
           <number>1638</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ChorusDepthLabel" >
          <property name="text" >
           <string>Depth</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ChorusDepth" >
          <property name="maximum" >
           <number>16383</number>
          </property>
          <property name="singleStep" >
           <number>16</number>
          </property>
          <property name="pageStep" >
           <number>1638</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval" >
           <number>1638</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="ChorusLevelLabel" >
          <property name="text" >
           <string>Level</string>
          </property>
          <property name="alignment" >
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="ChorusLevel" >
          <property name="maximum" >
           <number>16383</number>
          </property>
          <property name="singleStep" >
           <number>16</number>
          </property>
          <property name="pageStep" >
           <number>1638</number>
          </property>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickInterval" >
           <number>1638</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QPushButton" name="Push" >
          <property name="text" >
           <string>Load</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Pop" >
          <property name="enabled" >
           <bool>false</bool>
          </property>
          <property name="text" >
           <string>Delete</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="dumpInfoButton" >
          <property name="text" >
           <string>Dump Info</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" >
    <widget class="QGroupBox" name="GainBox" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>&lt;html>&lt;head>&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; />&lt;/head>&lt;body style=&quot; white-space: pre-wrap; font-family:Sans Serif; font-weight:400; font-style:normal; text-decoration:none;&quot;>&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;>Gain&lt;/p>&lt;/body>&lt;/html></string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="Gain" >
        <property name="minimum" >
         <number>0</number>
        </property>
        <property name="maximum" >
         <number>127</number>
        </property>
        <property name="pageStep" >
         <number>5</number>
        </property>
        <property name="value" >
         <number>13</number>
        </property>
        <property name="tracking" >
         <bool>true</bool>
        </property>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickInterval" >
         <number>7</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0" >
    <widget class="QFrame" name="ReverbFrame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" >
      <property name="margin" >
       <number>8</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="Reverb" >
        <property name="text" >
         <string>Reverb</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QLabel" name="ReverbRoomSizeLabel" >
            <property name="text" >
             <string>Room Size</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ReverbDampingLabel" >
            <property name="text" >
             <string>Damping</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ReverbWidthLabel" >
            <property name="text" >
             <string>Width</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="ReverbLevelLabel" >
            <property name="text" >
             <string>Level</string>
            </property>
            <property name="alignment" >
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" >
          <property name="margin" >
           <number>0</number>
          </property>
          <property name="spacing" >
           <number>6</number>
          </property>
          <item>
           <widget class="QSlider" name="ReverbRoomSize" >
            <property name="maximum" >
             <number>16383</number>
            </property>
            <property name="singleStep" >
             <number>16</number>
            </property>
            <property name="pageStep" >
             <number>1638</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval" >
             <number>1638</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="ReverbDamping" >
            <property name="maximum" >
             <number>16383</number>
            </property>
            <property name="singleStep" >
             <number>16</number>
            </property>
            <property name="pageStep" >
             <number>1638</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval" >
             <number>1638</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="ReverbWidth" >
            <property name="maximum" >
             <number>16383</number>
            </property>
            <property name="singleStep" >
             <number>16</number>
            </property>
            <property name="pageStep" >
             <number>1638</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval" >
             <number>1638</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="ReverbLevel" >
            <property name="maximum" >
             <number>16383</number>
            </property>
            <property name="singleStep" >
             <number>16</number>
            </property>
            <property name="pageStep" >
             <number>1638</number>
            </property>
            <property name="orientation" >
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickInterval" >
             <number>1638</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
     <widget class="QWidget" name="widget_2" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>36</y>
        <width>260</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_3" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>59</y>
        <width>260</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_4" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>82</y>
        <width>260</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
     <widget class="QWidget" name="widget_5" >
      <property name="geometry" >
       <rect>
        <x>11</x>
        <y>105</y>
        <width>260</width>
        <height>17</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="3" margin="8" />
 <pixmapfunction></pixmapfunction>
 <resources/>
 <connections/>
</ui>
